static/app/components/platformPicker.tsx

```diff
@@ -142,7 +147,7 @@ class PlatformPicker extends React.Component<Props, State> {
                 e.stopPropagation();
               }}
               onClick={() => {
-                analytics('platformpicker.select_platform', {platform: platform.id});
+                trackAdvancedAnalyticsEvent('growth.platform_pick', {platform}, null);
```

Comment:
@roggenkemper I think we should maintain the `select_platform` part of the event name. And also, probably use `platform.id` still